src/commands/bg/collection/create.ts

```
@@ -4,12 +4,14 @@ import {
   ruleSet
 } from '@metaplex-foundation/mpl-core'
 import type { TransactionSignature } from '@metaplex-foundation/umi'
-import { generateSigner } from '@metaplex-foundation/umi'
 import { Flags } from '@oclif/core'
 import ora from 'ora'
 
 import { TransactionCommand } from '../../../TransactionCommand.js'
 import { generateExplorerUrl } from '../../../explorers.js'
+import { generateSigner } from '@metaplex-foundation/umi'
```

ESLint

[error] Expected "`@metaplex-foundation/umi`" to come before "../../../explorers.js"

(perfectionist/sort-imports)